src/rpc/blockchain.cpp

```diff
@@ -409,8 +419,11 @@ UniValue getblock(const UniValue& params, bool fHelp)
     uint256 hash(uint256S(strHash));
     CBlockIndex* pblockindex = NULL;
     bool fVerbose = true;
-    if (params.size() > 1)
+    bool fListTxns = true;
+    if (params.size() == 2)
```

Comment:
if you only check size == 2, verbose will be ignored if "getblock XXX true false"